packages/svelte-ux/src/routes/+layout.svelte

```diff
@@ -52,6 +52,16 @@
           Cancel: 'Annuler',
```

Comment:
Might be nice to go ahead and add `OK` translation since we already have Cancel, and start to sprinkle it about...

![image](https://github.com/techniq/svelte-ux/assets/177476/774a90eb-e445-43fc-ac64-f7bc7181ba40)

![image](https://github.com/techniq/svelte-ux/assets/177476/3d36dacc-cf44-452b-96e1-1e483d9404fb)

Reply:
Ok, I gave a shot at it. Let's see